samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -105,11 +111,29 @@ static const struct bt_mesh_comp2_record comp_rec[2] = {
 	.elem_offset_cnt = 1,
 	.elem_offset = cmp2_elem_offset2,
 	.data_len = 0
+	},
+	{
+	.id = BT_MESH_NLC_PROFILE_ID_OCCUPANCY_SENSOR,
+	.version.x = 1,
+	.version.y = 0,
+	.version.z = 0,
+	.elem_offset_cnt = 2,
```

Comment:
This looks weird to me. `cmp2_elem_offset_2` has length 1, why is count set to 2 here (and 3 below)?
`elem_offset_cnt` is according to spec the "Total number of element offsets in the
Element_Offset_List field"

Reply:
And also: why is `.id = ...OCCUPANCY_SENSOR` listed 3 times, while people count and motion is not mentioned?

Reply:
First comment: I must have misunderstood the intention of the variables. Will fix. Second comment: they are all occupancy sensor types. Should this be updated somewhere? What is your suggestion?

Reply:
Oh sorry, second comment can be ignored, they are of course all implementing the same profile and should have the same profile ID.

First comment: The way this should be is that `elem_offset_cnt` should always be 1, and you have to create `cmp2_elem_offset_3`, `cmp2_elem_offset_4`, etc. (Preferably rename them to indicate what the element does, in my opinion, like `cmp2_elem_offset_ambient_light`, `cmp2_elem_offset_presence`, etc.)